docs/rpc.md

```diff
@@ -86,17 +98,24 @@ and we're not accepting issues related to them.
 | `sendmany` | Not applicable to neo-go, see `claimgas` comment |
 | `sendtoaddress` | Not applicable to neo-go, see `claimgas` comment |
 
-#### Implementation notices
+### Extensions
 
-##### `invokefunction` and `invoke`
+Some additional extensions are implemented as a part of this RPC server.
 
-neo-go's implementation of `invokefunction` and `invoke` does not return `tx`
-field in the answer because that requires signing the transaction with some
-key in the server which doesn't fit the model of our node-client interactions.
-Lacking this signature the transaction is almost useless, so there is no point
-in returning it.
+#### Websocket server
 
-Both methods also don't currently support arrays in function parameters.
+This server accepts websocket connections on `$BASE_URL/ws` address. You can
```

Comment:
Maybe it worth mentioning URI scheme: `ws://$BASE_URL/ws`

Reply:
ACK